ssr/src/page/token/create/server_impl.rs

```diff
@@ -302,14 +302,11 @@ mod real_impl {
         create_sns: SnsInitPayload,
     ) -> Result<DeployedCdaoCanistersRes, ServerFnError> {
         // NSFW check
-        let nsfw_info = match create_sns.token_logo.clone() {
-            Some(logo) => nsfw::get_nsfw_info(logo)
-                .await
-                .map_err(|e| ServerFnError::new(format!("failed to get nsfw info {e:?}")))?,
-            None => NSFWInfo::default(),
-        };
-        if nsfw_info.csam_detected {
-            return Err(ServerFnError::new("logo rejected"));
+        let mut nsfw_info = NSFWInfo::default();
```

Comment:
It would be more pragmatic to use if let syntax instead of inferring if the variant is some and then consequently unwrapping it...

```rust
if let Some(token_logo) = create_sns.token_logo {
       nsfw::get_nsfw_info(token_logo.clone())
            .await
            .map_err(|e| ServerFnError::new(format!("failed to get nsfw info {e:?}")))?;
}
```

Reply:
done